src/classes/window/presenter.vala

```diff
@@ -272,6 +272,17 @@ namespace pdfpc.Window {
             this.button_press_event.connect(this.presentation_controller.button_press);
             this.scroll_event.connect(this.presentation_controller.scroll);
 
+            // resize the bottom text based on the window height
+            // (see http://stackoverflow.com/a/35237445/730138)
+            var bottom_text_css_provider = new Gtk.CssProvider();
+            Gtk.StyleContext.add_provider_for_screen(Gdk.Screen.get_default(),
+                bottom_text_css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```

Comment:
We load the global css file with PRIORITY_APPLICATION. The auto scaling should have a lower priority than this. Maybe use "STYLE_PROVIDER_PRIORITY_FALLBACK" instead?